### PR TITLE
New haiku by jemaz

### PR DESCRIPTION
**Pull request by Jemaz**
Hi!

I wrote a haiku. It's not very impressive as this repo tho, but this meme will!

meme

### CODE REVIEW

**Comment by judemathew777** [comment]
layout: haiku
title: jude7
author: judemathew7
test: false
---

I am Jude<br>
trying get the hapiness<br>
with a pull request<br>

File: _haikus/cyberpunk.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Cyberpunk
+author: Jemaz
+---
+
+SSH, jacking in, handshake <br>
```
Comment (ltagliaferri) [inline _haikus/cyberpunk.md]: Currently, this line has too many syllables, maybe try:

```
SSH, handshake <br>
```

File: _haikus/cyberpunk.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Cyberpunk streaming neon lights <br>
```
Comment (ltagliaferri) [inline _haikus/cyberpunk.md]: This line also has one syllable too many, you can amend with:

```
Cyberpunk streaming neon <br>
```

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request. The changes requested have not been addressed over the course of a few days, so we are closing this PR due to inactivity and are marking it as `invalid`. If you would like to try again, please open a new pull request.

File: _haikus/cyberpunk.md (hunk 2 above)
Comment (Jemaz) [reply _haikus/cyberpunk.md]: Sorry for the (extremely) slow reply. 
I will take your feedback and reopen a new pull request.
Thanks.

**Pull request closed without merging**